packages/nextjs/services/database/config/postgresClient.ts

```diff
@@ -63,6 +74,10 @@ const dbProxy = new Proxy(
       const db = getDb();
       return db[prop];
     },
+    has: (target, prop: keyof DbProxy) => {
```

Comment:
We had to add this so that we can do: 

`if("batch" in db)` check, otherwise it JS was failing because there was no method on proxy to do it, and it was resulting in false always.